apidoc/Titanium/UI/Attribute.yml

```diff
@@ -115,3 +124,119 @@ properties:
+    description: |
+      This property contains the minimum height in points that any line in the receiver will occupy, 
+      regardless of the font size or size of any attached graphic. This value must be nonnegative.
+    type: Number
+
+  - name: lineSpacing
+    summary: The distance in points between the bottom of one line fragment and the top of the next.
+    description: |
+      This value is always nonnegative. This value is included in the line fragment heights 
+      in the layout manager.
+    type: Number
+
+  - name: paragraphSpacing
+    summary: The space after the end of the paragraph.
+    description: |
+      This property contains the space (measured in points) added at the end of the paragraph 
+      to separate it from the following paragraph. This value must be nonnegative. The space 
+      between paragraphs is determined by adding the paragraphSpacing of previous paragraph and 
+      the paragraphSpacingBefore of current paragraph.
+    type: Number
+
+  - name: paragraphSpacingBefore
+    summary: The distance between the top of paragraph and the beginning of its text content.
+    description: |
+      This property contains the space (measured in points) between the top of paragraph and 
+      the beginning of its text content. The default value of this property is 0.0.
+    type: Number
+
+  - name: lineHeightMultiple
+    summary: The line height multiple.
+    description: |
+      The natural line height of the receiver is multiplied by this factor (if positive) 
+      before being constrained by minimum and maximum line height. The default value of 
+      this property is 0.0.
+    type: Number
+
+  - name: hyphenationFactor
+    summary: The threshold of paragraph for hyphenation.
+    description: |
+      Valid values lie between 0.0 and 1.0 inclusive. The default value is 0.0. Hyphenation 
+      is attempted when the ratio of the text width (as broken without hyphenation) to the 
+      width of the line fragment is less than the hyphenation factor. When the hyphenation 
+      factor of paragraph is 0.0, the ] hyphenation factor of layout manager is used instead. 
+      When both are 0.0, hyphenation is disabled. This property detects the user-selected 
+      language by examining the first item in preferredLanguages.
+    type: Number
+
+  - name: allowsDefaultTighteningForTruncation
+    summary: |
+      A Boolean value indicating whether the system may tighten inter-character spacing 
+      before truncating text.
+    description: |
+      When this property is set to YES, the system tries to reduce the space between characters 
+      before truncating characters. The system performs this tightening in cases where the text 
+      would not otherwise fit in the available space. The maximum amount of tightening performed 
+      by the system is dependent on the font, line width, and other factors. The default value of 
+      this property is NO.
+    type: Number
```

Comment:
The `allowsDefaultTighteningForTruncation` type should be boolean, not number.

And the description should use `true` and `false` instead of the Objective-C `YES` and `NO`.